### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by AntoinePrv**

### CODE REVIEW

**Comment by pavelzw** [comment]
Can we also get a powershell test?

File: .github/workflows/test_micro.mamba.pm.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      fail-fast: false
+      matrix:
+        include:
+          - {os: ubuntu-latest, shell: bash, shell-source-param: -i}
+          - {os: macos-latest, shell: zsh, shell-source-param: -i}
+          - {os: macos-latest, shell: bash, shell-source-param: -l}
+          - {os: windows-latest, shell: bash, shell-source-param: -l}
+    steps:
+      - name: Install Micromamba (${{ matrix.os }}, ${{ matrix.shell }})
+        shell: ${{ matrix.shell }}
```
Comment (pavelzw) [inline .github/workflows/test_micro.mamba.pm.yml:39]: I don't think jinja stuff works in `shell:`. Did you test this action?

Comment (pavelzw) [reply .github/workflows/test_micro.mamba.pm.yml:39]: image
looks like it's not working

Comment (AntoinePrv) [reply .github/workflows/test_micro.mamba.pm.yml:39]: I did not since the CI wouldn't launch :sweat_smile:

**Pull request merged**